code/modules/projectiles/guns/mods/mods.dm

```diff
@@ -134,8 +134,8 @@
 	..()
 	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
 	I.weapon_upgrades = list(
-		GUN_UPGRADE_DAMAGE_RADIATION = 5
-		)
+		GUN_UPGRADE_DAMAGE_RADIATION = 100
```

Comment:
This is big.

Reply:
Not as big as it appears to be, I tested it